renderer-macos/lib/Brisk_macos.re

```diff
@@ -1,8 +1,11 @@
 module Brisk = Brisk;
 
 module Cocoa = {
-  module BriskApplication = BriskApplication;
-  module BriskWindow = BriskWindow;
+  module Types = CocoaTypes;
+
+  module Application = BriskApplication;
+  module Window = BriskWindow;
+
   module BriskMenu = BriskMenu;
```

Comment:
I think same applies to Menu.

Reply:
I know but I already had a Menu module in the user-space app, so wasn't sure what to do. 😁